build/android/bytecode/java/org/brave/bytecode/BraveBookmarkPageClassAdapter.java

```diff
@@ -16,7 +16,7 @@ public BraveBookmarkPageClassAdapter(ClassVisitor visitor) {
         super(visitor);
 
         redirectConstructor(sBookmarkPageClassName, sBraveBookmarkPageClassName);
-        deleteField(sBraveBookmarkPageClassName, "mManager");
-        makeProtectedField(sBookmarkPageClassName, "mManager");
+        deleteField(sBraveBookmarkPageClassName, "mBookmarkManagerCoordinator");
```

Comment:
Could you also add check for this change in bytecode tests please.

Reply:
It is already added. It was my mistake that after changes from chromium side, I didn't update variable name in the bytecode class adapter.